docs/src/modules/components/ThemeContext.js

```diff
@@ -216,8 +217,7 @@ export function ThemeProvider(props) {
           color: brandingDesignTokens.palette.primary.main,
         },
         palette: {
-          ...brandingDesignTokens.palette,
-          ...paletteColors,
```

Comment:
This is the root cause. the whole `primary` which contains only `main` property replace the branding color (50-900) are gone. So it causes the crash in some part that tries to use `alpha(...500)`.

Reply:
Interesting, I wonder if we shouldn't push it further, by only updating the demos, leaving the rest of the docs unchanged. I have noticed a couple of cases that are weird:

- Partially colored links:
  <img width="272" alt="Screenshot 2022-01-10 at 23 42 00" src="https://user-images.githubusercontent.com/3165635/148850440-0b564bcf-2446-42b6-9866-a96cb332d069.png">
- Partially colored sidenav:
  <img width="226" alt="Screenshot 2022-01-10 at 23 42 35" src="https://user-images.githubusercontent.com/3165635/148850512-f8b6edfc-1aa4-40f1-ba1b-397e26938bc7.png">